code/modules/modular_computers/file_system/programs/command/card.dm

```diff
@@ -41,6 +41,7 @@
 		data["id_owner"] = id_card && id_card.registered_name ? id_card.registered_name : "-----"
 		data["id_name"] = id_card ? id_card.name : "-----"
 
+	data["command_support_jobs"] = format_jobs(command_support_positions)
```

Comment:
a problem with this is that it lets command (presumably) set ID cards to the accesses that the HRA, consular, and liaison have. e.g. you wouldn't want people to somehow get the ID access to their offices.
i'd appreciate suggestions on how to solve this.